exhibits-solr-conf.gemspec

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'exhibits-solr-conf/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'exhibits-solr-conf'
```

Comment:
are you saying the gem *name* should have underscores, Jessie?   we have sul-exhibits-template and exhibits-request as precedents in that case.